lk2nd/device/dts/msm8952/msm8937-motorola-cedric.dts

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
```

Comment:
Perhaps add a comment like this and suggest a quirk of building with a single dtb for now then?

https://github.com/msm8916-mainline/lk2nd/blob/145fd710570af873d0b25a77f1cfb7d00a6b590b/lk2nd/device/dts/msm8916/msm8939-htc-m8qlul.dts#L6-L11

Reply:
I have chosen to add `msm8937-motorola-cedric.dtb` to `rules.mk` and to filter with `LK2ND_DTBS`.

When using `LK2ND_QCDTBS="msm8937-motorola-cedric.dtb"` instead, `LK2ND_ADTBS=""` would also be needed, to get rid of `msm8937-mtp.dtb`.